src/VolleyManagement.Backend/VolleyManagement.Services/GameService.cs

```diff
@@ -379,15 +380,15 @@ private void ValidateSetsScore(Score setsScore, bool isTechnicalDefeat)
             }
         }
 
-        private void ValidateSetsScoreMatchesSetScores(Score setsScore, IList<Score> setScores)
+        private static void ValidateSetsScoreMatchesSetScores(Score setsScore, IList<Score> setScores)
         {
             if (!ResultValidation.AreSetScoresMatched(setsScore, setScores))
             {
                 throw new ArgumentException(Resources.GameResultSetsScoreNoMatchSetScores);
             }
         }
 
-        private void ValidateSetScoresValues(IList<Score> setScores, bool isTechnicalDefeat)
+        private static void ValidateSetScoresValues(IList<Score> setScores, bool isTechnicalDefeat)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 18 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3776)